org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/model/bean/Contacts.java

```diff
@@ -0,0 +1,336 @@
+/*******************************************************************************
+ * Debrief - the Open Source Maritime Analysis Application
+ * http://debrief.info
+ *
+ * (C) 2000-2020, Deep Blue C Technology Ltd
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the Eclipse Public License v1.0
+ * (http://www.eclipse.org/legal/epl-v10.html)
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *******************************************************************************/
+
+package org.mwc.debrief.pepys.model.bean;
+
+import java.util.Date;
+
+import MWC.GenericData.WorldLocation;
+
+public class Contacts implements AbstractBean, FilterableBean, TreeStructurable{
+
+	private int contact_id;
```

Comment:
@saulhidalgoaular - there are lots of fields here, and I fear we may have to change the file to match schema changes that don't relate to Debrief. So, I'd suggest only storing those that can be exploited by Debrief.  Aah, maybe you were waiting for me to tell you which ones :-D

* time
* bearing
* rel_bearing
* freq
* major
* minor
* orientation
* classification
* subject_id
* source_id

Aah, actually, subject and source would be better than subject_id and source_id.  I'll pass that task to the database chap.

* location
*